manifests/0000_90_kube-controller-manager-operator_05_alerts.yaml

```diff
@@ -39,6 +39,14 @@ spec:
           for: 15m
           labels:
             severity: critical
+        - alert: PodsUnableToDelete
+          annotations:
+            summary: Some pods are unable to be deleted.
+            description: Some pods have been unable to be deleted for last 24h (since their grace period for termination has been reached).
+          expr: |
+            count by (namespace) (time() - kube_pod_deletion_timestamp > 86400)
+          labels:
+            severity: warning
```

Comment:
is `for:` useful here? since we are just waiting when the expression reaches over 24h.

Reply:
Right, we don't need `for`, because we're handling the "yeah, this is not going away" inertia in the `expr`.